src/java.base/macosx/classes/sun/nio/ch/KQueuePoller.java

```diff
@@ -65,11 +70,11 @@ void implDeregister(int fdVal, boolean polled) {
 
     @Override
     int poll(int timeout) throws IOException {
-        int n = KQueue.poll(kqfd, address, maxEvents, timeout);
+        int n = KQueue.poll(kqfd, pollArray, maxEvents, timeout);
         int i = 0;
         while (i < n) {
-            long keventAddress = KQueue.getEvent(address, i);
-            int fdVal = KQueue.getDescriptor(keventAddress);
+            MemorySegment eventMS = KQueue.getEvent(pollArray, i);
```

Comment:
Nit: Can we call these variables `eventSegment`? My reptile part of the brain first read "event in milliseconds".

Reply:
Yes, I think rename this to kevent to keventSegemnt as it's a kevent struct (which is why the Unsafe usage named it keventAddres).